pkg/didcomm/dispatcher/dispatcher.go

```diff
@@ -21,4 +23,8 @@ type DIDCommMsg struct {
 	Outbound bool
 	Type     string
 	Payload  []byte
+	//send: Function to send outbound messages
+	Send func(interface{}, string, string) error
+	// NewDidDoc Function to send outbound exchange request messages
+	NewDidDoc func() (*did.Doc, error)
```

Comment:
Should this be part of the DIDCommMsg struct ? This looks specific to DIDExchange protocol. This might be irrelevant to other protocols.

Reply:
@rolsonquadras changes to DIDCommMsg will be reverted in the next update

Reply:
@rolsonquadras i am working on the patch set to update this

Reply:
done